Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "MainWidget.h"
 #include <AK/Optional.h>
+#include <AK/Random.h>
```

Comment:
It doesn't look like this is used at all...

Reply:
Thank you :), leftovers from when I was trying to use a token approach.